jobs/build/kn_sync/Jenkinsfile

```diff
@@ -1,82 +0,0 @@
-#!/usr/bin/env groovy
-node {
-    checkout scm
-    commonlib = load("pipeline-scripts/commonlib.groovy")
-    commonlib.describeJob("kn_sync", """
-        ----------------------------------------------
-        Sync the knative (serverless) client to mirror
-        ----------------------------------------------
-        http://mirror.openshift.com/pub/openshift-v4/x86_64/clients/serverless/
-
-        Timing: This is only ever run by humans, upon request.
-    """)
-}
-
-pipeline {
-    agent any
-    options { disableResume() }
-
-    parameters {
-        string(
-            name: "VERSION",
-            description: "Desired version name. Example: 0.20.0",
-            defaultValue: "",
-            trim: true,
-        )
-        string(
-            name: "SOURCES_LOCATION",
-            description: "Example: http://download.eng.bos.redhat.com/staging-cds/developer/openshift-serverless-clients/0.20.0-6/signed/all/",
-            defaultValue: "",
-            trim: true,
-        )
-        booleanParam(
-            name: 'SKIP_ARM64',
-            description: 'Skip mirroring arm64 binary.',
-            defaultValue: false,
-        )
-    }
-
-    stages {
-        stage("Validate params") {
-            steps {
-                script {
-                    if (!params.VERSION) {
-                        error "VERSION must be specified"
-                    }
-                    if (params.SOURCES_LOCATION[-1] != "/") {
-                        error "Location should end with a trailing slash (to not confuse wget)"
-                    }
-                }
-            }
-        }
-        stage("Clean working dir") {
-            steps {
-                sh "rm -rf ${params.VERSION}"
-            }
-        }
-        stage("Download binaries") {
-            steps {
-                script {
-                    downloadRecursive(params.SOURCES_LOCATION, params.VERSION, params.SKIP_ARM64)
-                }
-            }
-        }
-        stage("Sync to mirror") {
-            steps {
-                sh "tree ${params.VERSION}"
-                script {
-                    commonlib.syncDirToS3Mirror("${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/serverless/${params.VERSION}/")
-                    commonlib.syncDirToS3Mirror("${params.VERSION}/", "/pub/openshift-v4/x86_64/clients/serverless/latest/")
-                }
-            }
-        }
-    }
-}
-
-def downloadRecursive(path, destination, skip_arm64) {
-    skip_param = ""
-    if (skip_arm64) {
```

Comment:
Something tells me kn_sync didn't want arm published.. at some point anyways. Maybe we can have a generic skipArch param and func in client_sync to get this flexibility, What do you think?